Firmware/Marlin_main.cpp

```diff
@@ -6512,6 +6513,31 @@ SERIAL_PROTOCOLPGM("\n\n");
     }
     break;
 
+    /*!
+    ### M118 - Serial print <a href="https://reprap.org/wiki/G-code#M118:_Echo_message_on_host">M118: Serial print</a>
+    */
```

Comment:
Since we are handling E and A parameters. I think we ought to document what they do (taken from here https://marlinfw.org/docs/gcode/M118.html). You can see a similar format used for other gcodes in `Marlin_main.cpp`.

### M118 - 
    #### Usage
    
        M118 [ A1 | E1 ] [ String ]
    
    #### Parameters
    - `A1` - Prepend // to denote a comment or action command. Hosts like OctoPrint can interpret such commands to perform special actions. See your host’s documentation.
    - `E1` - Prepend echo: to the message. Some hosts will display echo messages differently when preceded by echo:.
    - `String` - Message string. If omitted, a blank line will be sent.
    */

Reply:
Thanks for the quick feedback. 
Indeed adding usage information to the comment was something I overlooked. Added now ;)